handbook/data-stores/file-storage.md

```diff
@@ -0,0 +1,3 @@
+# File Storage
+
+Files ingested by the SystemLink File Service by...
```

Comment:
I think you already provided some context on this... but is this the final form of this file?

Reply:
Same q. Based on a previous thread it seems like this will be a short section potentially linking out to SPD.

Reply:
I've removed this file for now. While we won't go live with multiple subchapters I want to have this structure now because changing it will affect URLs.